package.json

```diff
@@ -13,19 +13,25 @@
     "prebuild": "npm run clean",
     "test": "./test/run_tests_and_server.sh",
     "dev": "webpack --progress --colors --config webpack.config.js --mode=development",
-    "build": "webpack --progress --colors --config webpack.config.js --mode=production",
+    "build": "webpack --progress --colors --config webpack.config.js --mode=production && cp dist/webgazer.js www/.",
     "gen_docs": "jsdoc -c jsdoc.conf.json src/*"
```

Comment:
If want to go back to the old Website generated docs, we could add one_docs to the commit hooks as well.

Reply:
You also want to cp to www/data/src so that webgazerExtract works